packages/react-text/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-text",
-  "version": "0.3.3",
-  "description": "Fluent UI React Text component.",
+  "version": "9.0.0-alpha.0",
```

Comment:
also I'd mark this as private until we ship 1st real alpha implementation ?

Reply:
`@fluentui/react-text` is already released unfortunately https://www.npmjs.com/package/@fluentui/react-text

If `private` will still work in that case, yeah I agree with doing it

Reply:
it should, `private` will just do nothing for beachball thus no npm publish will happen